### DISCUSSION TITLE
通过邮箱授权收不到恢复码!

### DISCUSSION THREAD

**Discussion by piano885** [getlantern/lantern-forum-cn#461] [Q&A]
上个月开始就闪退,下了新版本就登录不上了
17149956@qq.com
我已经找不到支付凭证了.

**Comment by MaxMKMax** [comment]
@piano885 您好. 我会帮您提交问题然后给您发送恢复码. 也请您提供我付款单据. 谢谢! 请您过两天查看邮箱或Github. 谢谢!

**Reply by piano885** [reply]
好的

**Reply by piano885** [reply]
这没后续了?

**Reply by MaxMKMax** [reply]
@piano885 您好. 支付团队那里要求支付单据的. 您可以试试看找一下吗? 抱歉, 麻烦您了

**Reply by piano885** [reply]
我一次买了两年的根本找不到了,这样就是说你们后台已经没有这个邮箱的购买记录然后就赖到过期就没了?

**Reply by MaxMKMax** [reply]
@piano885 您好. 那我来让他们从后台找找看! 我们不会赖到过期的. 请您放心! 我们之后会对您无法使用的时间给您补偿. 重新延长您的专业版. 有什么不好的体验我们向您说声抱歉. 谢谢!

**Reply by piano885** [reply]
总之,我现在提供不了支付单据,买的两年都一年以前的单据上哪找?根本找不到.
现在怎么办?就一直拖?你至少说个时间:几个工作日内解决

**Reply by MaxMKMax** [reply]
@piano885 您好. 给我们 3 个工作日解决. 快的话 2 天后会给您发邮件或在这里给您留言. 非常抱歉让您久等了!

**Comment by MaxMKMax** [comment]
@piano885 您好. 已经给您发激活码了. 请查看邮箱. 标题: 蓝灯激活码

谢谢您耐心的等待. 非常抱歉让您久等了!

**Reply by piano885** [reply]
能解决问题就可以.也感谢你的服务

**Reply by MaxMKMax** [reply]
@piano885 您好. 不客气呢. 能帮您解决到问题我非常开心. 谢谢您支持蓝灯. 感激不尽!

**Comment by LXabc123** [comment]
你好，我的邮箱收不到恢复码，请帮忙解决一下

**Reply by MaxMKMax** [reply]
@LXabc123 您好. 请您提供我您的账号邮箱和支付单据. 我帮您提交问题. 谢谢.

**Reply by LXabc123** [reply]
![微信图片_20240613174346](https://github.com/getlantern/lantern-forum-cn/assets/163421346/64f4a751-2054-4466-8275-23d584db468f)
邮箱：dongguirong2005@xaut.edu.cn

**Reply by MaxMKMax** [reply]
@LXabc123 
您好. 我已经成功帮您提交问题了. 请您给我们几天工作日解决. 下周二应该好了.

**Comment by MaxMKMax** [comment]
@LXabc123 
您好. 这是您的恢复码 激活码. 不好意思让您久等了.
7YYG2-99MK7-C949B-3R3F4-44HYB
